### DISCUSSION TITLE
bun run start的时候执行--update-cqps是不是会出问题

### DISCUSSION THREAD

**Discussion by Miuzarte** [Tsuk1ko/CQ-picfinder-robot#464] [一般讨论]
今天--update-cqps了一下，直到刚刚发现一直有两个cqps同时在跑
一个在bun上一个在node上
但是我旧的cqps是用的bun run test跑的，所以没法确定bun run start或者npm run bun会不会没问题

至于为啥要run test，之前突然开始用node v18, npm run start丢后台跑一段时间会直接没反应，语料库和搜图都是

用npm run test挂前台倒是没出现过，不知道是我termux proot容器的锅还是nodev18的锅，换了bun再bun run start还是一样

后面换node20再试试好了

常开debug别人又没法用，太难了

**Comment by Tsuk1ko** [comment]
bun 运行的没有适配自动更新，不要一键更新

**Comment by Tsuk1ko** [comment]
并且一键更新是按你用 `npm start` 启动的场景来做的，即用 pm2 的场景，前台启动的也不要用这个自动更新
